plutus-pab/src/Plutus/PAB/Run/PSGenerator.hs

```diff
@@ -47,6 +51,9 @@ import           Servant.PureScript                         (HasBridge, Settings
                                                              defaultBridge, defaultSettings, languageBridge,
                                                              writeAPIModuleWithSettings)
 
+class HasPSTypes a where
+  psTypes :: Proxy a -> [SumType 'Haskell]
```

Comment:
Haddocks, we could also add a default definition that returns the empty list (I'm assuming that some (most?) clients won't be written in Purescript so they don't need the PS bridge)

Reply:
Done. Added default implement. Haddocks adequate?